src/main/java/practice/day5/homework/School.java

```diff
@@ -0,0 +1,157 @@
+package practice.day5.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class School {
+    private ArrayList<Student> students;
+    private ArrayList<Teacher> teachers;
+
+    public School() {
+        this.students = new ArrayList<>();
+        this.teachers = new ArrayList<>();
+    }
+
+    public void addStudent(Student student) {
+        students.add(student);
+    }
+
+    public void addTeacher(Teacher teacher) {
+        teachers.add(teacher);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
You don't need to override toString() here if you aren't going to return information about the School class. By writing super.toString() you are calling the method defined in class Object. If toString() isn't overridden, when we use toString(), the method in the Object class will be called. Remember that every class extends Object.